transcribe/scribe_modules/k8s_pods.py

```diff
@@ -1,5 +1,5 @@
 from . import ScribeModuleBaseClass
-from . lib.util import format_url
+from . lib.util import to_list
```

Comment:
Instead of importing this, this particular module requires following definition:

```
def to_list(first_lvl, second_lvl, my_dict):
    if first_lvl in my_dict:
        if second_lvl in my_dict[first_lvl]:
            new_dict = {}
            for key,value in my_dict[first_lvl][second_lvl].items():
                if '.' in key:
                    new_dict[key.replace('.','_')] = value
            my_dict[first_lvl][second_lvl] = new_dict
            tmp_list = []
            tmp_list.append(my_dict[first_lvl][second_lvl])
            my_dict[first_lvl][second_lvl] = tmp_list
    return my_dict
```